ui/admin/setup-edit.php

```diff
@@ -168,7 +168,7 @@
                     ),
                     'class' => 'pods-validate pods-validate-required'
                 ) ); ?>
-                    <input type="button" class="save-button button" value="OK" /> <a class="cancel" href="#cancel-edit">Cancel</a>
+                    <input type="button" class="save-button button" value="<?php _e( 'OK', 'pods' ); ?>" /> <a class="cancel" href="#cancel-edit"><?php _e( 'Cancel', 'pods' ); ?></a>
```

Comment:
Hi @sc0ttkclark 
Just pushed it in https://github.com/pedro-mendonca/pods/commit/303501e49814329055d1231bdc83cf9702e09a31
Is this correct?